release/maven-central/heron-api-0.14.0.pom

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we commit this generically as heron-api.pom.template without placeholder tokens for version and include a bash script to generate the pom given a version? That way we don't need to edit, rename and recommit every time we release. Same for the other pom.